corehq/apps/cloudcare/static/cloudcare/less/markdown-table.less

```diff
@@ -36,4 +36,10 @@
             }
         }
     }
-}
\ No newline at end of file
+}
+
+.text-center {
```

Comment:
@orangejenny The nesting is reverse from your first comment on the story because this is how they appear on the page. But this works now.